plugins/modules/proxmox.py

```diff
@@ -51,6 +52,41 @@
       - See U(https://pve.proxmox.com/wiki/Linux_Container) for a full description.
       - Should not be used in conjunction with O(storage).
     type: str
+  disk_volume:
+    description:
+      - Specify a hash/dictionary of the C(rootfs) disk.
+      - See U(https://pve.proxmox.com/wiki/Linux_Container#pct_mount_points) for a full description.
+    type: dict
+    suboptions:
+      storage:
+        description:
+          - V(storage) is the storage identifier of the storage to use for the C(rootfs).
+          - Mutually exclusive with V(host_path).
+        type: str
+      volume:
+        description:
+          - V(volume) is the name of an existing volume.
+          - If not defined, the module will check if one exists. If not, a new volume will be created.
+          - If defined, the volume must exist under that name.
+          - Required only if V(storage) is defined and mutually exclusive with V(host_path).
+        type: str
+      size:
+        description:
+          - V(size) is the size of the storage to use.
+          - The size is given in GB.
+          - Required only if V(storage) is defined and mutually exclusive with V(host_path).
+        type: int
+      host_path:
+        description:
+          - V(host_path) defines a bind or device path on the PVE host to use for the C(rootfs).
+          - Mutually exclusive with V(storage), V(volume), and V(size).
+        type: path
+      options:
+        description:
+          - V(options) is a list of extra options.
+          - The options are given as key-value pairs such as C([ro=0, acl=1]).
```

Comment:
Shouldn't this be a dictionary then? Or is the order important, or is it possible to have keys without values?

Reply:
This could be a dictionary, correct. Though then it would be slightly more work converting it to the correct string. I'll see if I can change it quickly.

Reply:
The hardest part for this was getting the documentation right.